package.json

```diff
@@ -12,7 +12,8 @@
     "react-dom": "^16.7.0",
     "react-hamburger-menu": "^1.1.1",
     "react-router-dom": "^4.3.1",
-    "react-scripts": "^2.1.5"
+    "react-scripts": "^2.1.5",
+    "validator": "^10.11.0"
```

Comment:
I don't see anywhere in this PR where validator is actually getting called/used. Do we need to have it imported?